pkg/mutators/azureasomanagedcontrolplane.go

```diff
@@ -350,12 +356,14 @@ func setAgentPoolProfilesFromAgentPools(managedCluster conversion.Convertible, a
 			ScaleDownMode:                     hubPool.Spec.ScaleDownMode,
 			ScaleSetEvictionPolicy:            hubPool.Spec.ScaleSetEvictionPolicy,
 			ScaleSetPriority:                  hubPool.Spec.ScaleSetPriority,
+			SecurityProfile:                   hubPool.Spec.SecurityProfile,
```

Comment:
The main alternative to this approach is to use unkeyed struct fields and let the compiler check that everything is defined, but I think I prefer the sanity check of having both field names on one line like this.